Makefile

```diff
@@ -96,19 +96,22 @@ test-go: ## Run Golang tests
 	@go test ./...
 
 test-alipay: ## Run tests for Alipay provider
```

Comment:
`test-alipay-beancount` and `test-alipay-ledger` or just `test-alipay`?

I prefer the first ones, what do u guys think? /cc @gaocegege @ramsayleung

Reply:
Sure, it makes sense to me either.